### ISSUE TITLE
Error when creating a New Resource 

### ISSUE THREAD

**Issue by katiagbif** [gbif/ipt#2097]
IPT (https://ipt.gbif.es/ Integrated Publishing Toolkit (IPT) Version 3.0-RC1) returns an error when creating a new resource. 
![image](https://github.com/gbif/ipt/assets/7556258/45e0e529-30da-4428-adb0-bdaf16e96454)

The resource can't  be created.

See attached logfile

[logfile_ipt_20230911.txt](https://github.com/gbif/ipt/files/12575507/logfile_ipt_20230911.txt)

Thanks!

**Comment by mike-podolskiy90** [comment]
@katiagbif Thank you for contacting us
Why are you using the IPT 3.0-RC1? It's not for production use, it's a release candidate only for testing

**Comment by katiagbif** [comment]
oops! We had installed the previous version, we don't know what happened. We will reinstall it again and I'm sure it will be fixed. Thank you very much for the warning! :)

**Comment by mike-podolskiy90** [comment]
If you're using auto-update system, please make sure you update to the latest release version (not pre-release) https://github.com/gbif/ipt/releases. The latest release is 2.7.5

**Comment by vjrj** [comment]
This is my fault. I updated my debian PR https://github.com/gbif/ipt/pull/1470 to `3.0-RC1` and I generated a local deb package to test it in https://ipt-demo.gbif.es . So I copied there, and I installed it manually.

But, I did a push of my code :-( , so our CI created the package automatically, published the package to our debian repository `apt.gbif.es` and our production server installed it automatically. I always trigger the build manually, so I didn't realized that I have configured the auto track and build of the repository.

I've just downgraded https://ipt.gbif.es, but I hope that others does not auto update the package too (I checked other ipt deployed via our apt.gbif.es repo that I know and they are not affected).

I removed the pre-release package also from our apt repository. Sorry!
